src/actions/employeesActions.js

```diff
@@ -59,6 +60,24 @@ export const updateSkypeResult = (resultBlock, loading) => {
   };
 };
 
+export const getEmployeeBySkill = employeesBySkill => {
+  return {
+    type: GET_EMPLOYEES_BY_SKILL,
+    employeesBySkill
+  };
+};
+
+export const getEmployeesBySkillACreator = (skillId) => {
```

Comment:
You can avoid return statement and curly brackets like that. 
![image](https://user-images.githubusercontent.com/22937810/51467802-b2357780-1d6d-11e9-981c-6b4e7deeeb74.png) and 
![image](https://user-images.githubusercontent.com/22937810/51467814-b95c8580-1d6d-11e9-8953-709b8a8c86af.png)